src/js/edu-benefits/1995/helpers.jsx

```diff
@@ -23,7 +24,16 @@ export const preferredContactMethodLabels = {
 };
 
 export function transform(formConfig, form) {
-  const formData = transformForSubmit(formConfig, form);
+  // All the king's horses and all the king's men
+  //  Put newSchool back together again.
+  const repairedForm = _.set('newSchool.data.newSchool', {
```

Comment:
I didn't look to see that tests had been written for this at all yet...

Just modified the data for the unit tests to mirror what's happening in the front end with `newSchool`, so the transform does have to piece it together in the unit test now.